test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/LineMappingsSerializer.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text;
+using Microsoft.AspNetCore.Razor.Evolution;
+
+namespace Microsoft.AspNetCore.Mvc.Razor
+{
+    public static class LineMappingsSerializer
```

Comment:
Is this copy-paste from Razor? Should we make the integration testing stuff we built in razor into its own package so we can use it here?